README.md

```diff
@@ -72,7 +74,7 @@ You never need to guess, if you're ever curious where you are or need to confirm
 
 ## `code` - Your Home Directory
 
-When you signed up for Learn, we created a new workspace for you (labeled with your Github username) and a default directory called `code`, which is where all of your code will live. 
+When you signed up for Learn, we created a new workspace for you (labeled with your Github username) and a default directory called `code`, which is where all of your code will live.
```

Comment:
I think this is a tarpit.

On the sandbox i'm put in `/Users/sgharms/Development`; but if i type `cd` i'm in `/Users/sgharms`

Labs do differently and put their content in `/Users/sgharms/labname`

I don't see this `code` thing used.

Can we remove the interface-specificity here?